docs/pages/access-controls/getting-started.mdx

```diff
@@ -40,30 +29,29 @@ Members of `admin`, are full cluster administrators.
 
 Invite a local user Alice as cluster `editor`:
 
-```bash
-tctl users add alice --roles=editor
+```code
+$ tctl users add alice --roles=editor
 ```
 
 Once Alice signs up, she will be able to edit cluster configuration. You can list
 users and their roles using `tctl users ls`.
 
-```bash
-tctl users ls
+```code
```

Comment:
I'm curious, with this `code` block type, how does it know which language's syntax highlighting to apply? Is this only for shell? Or does it autodetect somehow?

Reply:
Came here to ask the same question. Do you have a reference for what the "code" block means to our particular renderers?

Adding to the above, won't the `$` interfere when copying/pasting snippets?

Reply:
@codingllama @r0mant `code` block is not for general purpose programming language, it's mostly for bash code snippets, as it's very specific in it's behavior with `$` and `#`

Reply:
@codingllama the plugin handles `$` and understands line continuation

Reply:
Nice, thanks.